lib/hydra/derivatives/processors/full_text.rb

```diff
@@ -50,11 +50,12 @@ def original_size
 
       # @returns [URI] path to the extract service
       def uri
-        @uri ||= URI("#{connection_url}/update/extract?extractOnly=true&wt=json&extractFormat=text")
+        @uri ||= connection_url + 'update/extract?extractOnly=true&wt=json&extractFormat=text'
```

Comment:
Why the change from string interpolation to appending? Don't have a strong objection -- mostly curious.